<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page100</class>
 <widget class="QWidget" name="page100">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>page100</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(125,125,125);
color: rgb(255, 255, 0);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="ATCMbutton" name="atcmButton300">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>ATCMbutton widget</string>
     </property>
     <property name="pageName">
      <string notr="true">page300</string>
     </property>
     <property name="statusvar">
      <string notr="true"/>
     </property>
     <property name="viewStatus">
      <bool>false</bool>
     </property>
     <property name="visibilityVar">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>300 Super</string>
     </property>
     <property name="pressText">
      <string>300 Super</string>
     </property>
     <property name="borderWidth">
      <number>1</number>
     </property>
     <property name="borderRadius">
      <number>0</number>
     </property>
     <property name="apparence">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>page100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="ATCMbutton" name="atcmButton200">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>ATCMbutton widget</string>
     </property>
     <property name="pageName">
      <string notr="true">page200</string>
     </property>
     <property name="statusvar">
      <string notr="true"/>
     </property>
     <property name="viewStatus">
      <bool>false</bool>
     </property>
     <property name="visibilityVar">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>200 User</string>
     </property>
     <property name="pressText">
      <string>200 User</string>
     </property>
     <property name="borderWidth">
      <number>1</number>
     </property>
     <property name="borderRadius">
      <number>0</number>
     </property>
     <property name="apparence">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ATCMbutton" name="atcmButton400">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>ATCMbutton widget</string>
     </property>
     <property name="pageName">
      <string notr="true">page400</string>
     </property>
     <property name="statusvar">
      <string notr="true"/>
     </property>
     <property name="viewStatus">
      <bool>false</bool>
     </property>
     <property name="visibilityVar">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>400 Admin</string>
     </property>
     <property name="pressText">
      <string>400 Admin</string>
     </property>
     <property name="borderWidth">
      <number>1</number>
     </property>
     <property name="borderRadius">
      <number>0</number>
     </property>
     <property name="apparence">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ATCMbutton" name="atcmButtonMenu">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>ATCMbutton widget</string>
     </property>
     <property name="pageName">
      <string notr="true">menu</string>
     </property>
     <property name="statusvar">
      <string notr="true"/>
     </property>
     <property name="viewStatus">
      <bool>false</bool>
     </property>
     <property name="visibilityVar">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
     <property name="pressText">
      <string>Menu</string>
     </property>
     <property name="borderWidth">
      <number>1</number>
     </property>
     <property name="borderRadius">
      <number>0</number>
     </property>
     <property name="apparence">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="ATCMbutton" name="atcmButtonPassword">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>ATCMbutton widget</string>
     </property>
     <property name="pageName">
      <string notr="true">login_pwd</string>
     </property>
     <property name="statusvar">
      <string notr="true"/>
     </property>
     <property name="viewStatus">
      <bool>false</bool>
     </property>
     <property name="visibilityVar">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>login_pwd</string>
     </property>
     <property name="pressText">
      <string>login_pwd</string>
     </property>
     <property name="borderWidth">
      <number>1</number>
     </property>
     <property name="borderRadius">
      <number>0</number>
     </property>
     <property name="apparence">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblProtection">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblPasswordLevel">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lblUser">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="lblSuper">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lblAdmin">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="ATCMbutton" name="atcmButtonSetPwd">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string>ATCMbutton widget</string>
     </property>
     <property name="pageName">
      <string notr="true">set_password</string>
     </property>
     <property name="statusvar">
      <string notr="true"/>
     </property>
     <property name="viewStatus">
      <bool>false</bool>
     </property>
     <property name="visibilityVar">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>set_pwd</string>
     </property>
     <property name="pressText">
      <string>set_pwd</string>
     </property>
     <property name="borderWidth">
      <number>1</number>
     </property>
     <property name="borderRadius">
      <number>0</number>
     </property>
     <property name="apparence">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblLevel">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblMSVersion">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="lblModel">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ATCMbutton</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>atcmButton300</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>page100</receiver>
   <slot>goto_page(const char*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>atcmButton200</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>page100</receiver>
   <slot>goto_page(const char*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>atcmButton400</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>page100</receiver>
   <slot>goto_page(const char*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>atcmButtonMenu</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>page100</receiver>
   <slot>goto_page(const char*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>atcmButtonPassword</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>page100</receiver>
   <slot>goto_page(const char*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>atcmButtonSetPwd</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>page100</receiver>
   <slot>goto_page(const char*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goto_page(const char*,bool)</slot>
 </slots>
</ui>
